sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>[Grammar]</p><p><s>and hence faster</s> --&gt; <code>and hence, faster</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>